string[] uav_ids
geometry_msgs/Point[] positions
float64 min_x
float64 max_x
float64 min_y
float64 max_y